openssh.yaml

```
@@ -129,3 +121,13 @@ subpackages:
     dependencies:
       runtime:
         - openssh-keygen
+
+advisories:
+  CVE-2023-25136:
+    - timestamp: 2023-02-06T13:39:35.322093-05:00
+      status: fixed
+      fixed-version: 9.0_p1-r0
```

This CVE was fixed in 9.2, not 9.0.

```suggestion
      fixed-version: 9.2_p1-r0
```

---

Fixed!